app/src/main/java/com/practicum/playlistmaker/PlayerActivity.kt

```diff
@@ -1,42 +1,134 @@
 package com.practicum.playlistmaker
 
+import android.icu.text.SimpleDateFormat
+import android.media.MediaPlayer
 import android.os.Bundle
+import android.os.Handler
+import android.os.Looper
 import androidx.appcompat.app.AppCompatActivity
 import com.bumptech.glide.Glide
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners
 import com.practicum.playlistmaker.databinding.ActivityAudioPlayerBinding
+import java.util.Locale
 
 class PlayerActivity : AppCompatActivity() {
 
     private lateinit var track: Track
     private lateinit var binding: ActivityAudioPlayerBinding
+    private var playerState = STATE_DEFAULT
+    private var mediaPlayer = MediaPlayer()
```

Comment:
```suggestion
    private val mediaPlayer = MediaPlayer()
```
Тут скорее всего `val`, объект плеера же не пересоздаётся